pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -254,7 +266,7 @@ func (a Adapter) Push(parameters common.PushParameters) (err error) {
 
 	a.deployment, err = a.Client.WaitForDeploymentRollout(a.deployment.Name)
 	if err != nil {
-		return errors.Wrap(err, "error while waiting for deployment rollout")
+		return errors.Wrapf(err, "Failed to update config to component deployed.")
```

Comment:
There's a separate PR for removing the occurrences of `errors.Wrapf`, but wherever you are modifying things, can you make sure to use `fmt.Errorf`? Otherwise, I think, we will end up having `errors.Wrapf` occurrences if that PR goes in before yours.

Reply:
@feloy WDYT? I see you already `lgtm`'d this PR. Since you are working on the other PR, I am OK by adding LGTM back to this PR if you would prefer this one to go in first.

Reply:
We will turn @cdrage into a weather vane (is this phrase existing in english?), as he already changed this back and forth several times depending on different reviews :( 
I created the PR https://github.com/redhat-developer/odo/pull/5557 to resolve the misunderstanding, let's keep it as is here

Reply:
/lgtm